api/app/routers/fba.py

```diff
@@ -1,14 +1,17 @@
 """ Routers for new/formal/non-tinker fba.
```

Comment:
```suggestion
""" Routers for Auto Spatial Advisory
```

Reply:
Ugh there's too many references to "fba" etc. I'm giving up on this now. It's beyond the scope of this ticket anyway.